tests/sweep_framework/sweeps/conv2d/short/conv2d_short_sweep.py

```diff
@@ -0,0 +1,448 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Optional, Tuple, List
+import os
+import itertools
+import random
+import torch
+
+import ttnn
+
+from tests.ttnn.utils_for_testing import check_with_pcc, start_measuring_time, stop_measuring_time
+from models.utility_functions import torch_random
+from tests.sweep_framework.sweeps.conv2d.short.conv2d_common import run, mesh_device_fixture
+
+parameters = {
+    "short_sweep_suite": {
+        "input_specs": [
+            [1, 32, 1, 28, 28, 3, 3, 1, 1, 0, 0, 1, True, 1],
```

Comment:
Can you add a comment line with the names of each of the params in the input_specs

Reply:
Done.